### ISSUE TITLE
Plot track in flight reports

### ISSUE THREAD

**Issue by lkluft** [orcestra-campaign/book#489]
Since both the aircraft position and the GOES satellite image are freely available, we should consider automatically rendering the flight track in the flight reports instead of including it as a static asset.

**Comment by bjorn-stevens** [comment]
I agree.. is there a template that one could use to do this for all flights?  Should we do this as video?  The only downside is that the download of the GOES is heavy.

**Comment by lkluft** [comment]
Creating the video is definitely too heavy for a function that runs during the build. However, we could include a script to create and insert the flight tracks. I guess that the challenging part is to make the data available in the CI (see #491)

**Comment by Lucalino** [comment]
Should this issue be closed? At this point, most flight reports contain an animation of the flight tracks.

**Comment by lkluft** [comment]
I will close this issue for now as most flight reports actually include a plot of the flight track **and** an animation that shows the trajectory in time.

For future reference: it now is possible to access IPFS data from the CI/CD. Therefore, it would actually be possible to render flights tracks on-demand in the reports.
